### ISSUE TITLE
Updating seems broken

### ISSUE THREAD

**Issue by zoidby** [woefe/ytcc#71]
Since yesterday (2020-11-09) updating doesn’t add new videos despite multiple of my subscriptions published new videos since.

**Comment by hpsu** [comment]
Can confirm. No updates for 3 days. `ytcc -l debug update` shows no errors

**Comment by EmRowlands** [comment]
Can you post the output of `ytcc bug-report`?

**Comment by zoidby** [comment]
```
> ytcc bug-report
---ytcc version---
2.0.1

---youtube-dl version---
2020.11.01.1

---SQLite version---
SQLite system library version: 3.33.0
Python module version: 2.6.0

---python version---
3.8.6 (default, Sep 30 2020, 04:00:38) 
[GCC 10.2.0]

---mpv version---
mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       56.51.100
   libavcodec      58.91.100
   libavformat     58.45.100
   libswscale      5.7.100
   libavfilter     7.85.100
   libswresample   3.7.100
ffmpeg version: n4.3.1


---config dump---
[ytcc]
download_dir = ~/pub
mpv_flags = --ytdl-format=bestvideo[height<=?1080]+bestaudio/best --geometry=1272
order_by = playlists:asc, publish_date:asc
video_attrs = playlists, title, duration
playlist_attrs = name, url, tags
db_path = ~/.local/share/ytcc/ytcc.db
date_format = %Y-%m-%d
max_update_fail = 10
max_update_backlog = 21
age_limit = 9000

[tui]
alphabet = sdfervghnuiojkl
default_action = play_video

[theme]
prompt_download_audio = 51
prompt_download_video = 46
prompt_play_audio = 51
prompt_play_video = 46
prompt_mark_watched = 196
table_alternate_background = 168

[youtube_dl]
format = bestvideo[height<=?1080]+bestaudio/best
output_template = %(title)s.%(ext)s
ratelimit = 0
retries = 0
subtitles = off
thumbnail = true
skip_live_stream = true
merge_output_format = mkv
```

**Comment by EmRowlands** [comment]
Appears to be a youtube-dl upstream bug:
```
$ youtube-dl -J 'https://youtube.com/channel/UC9infsKo33_2LUoiqXGgQWg'
{"_type": "playlist", "entries": [], "id": "UU9infsKo33_2LUoiqXGgQWg", "uploader": null, "uploader_id": null, "uploader_url": null, "extractor": "youtube:playlist", "webpage_url": "https://www.youtube.com/playlist?list=UU9infsKo33_2LUoiqXGgQWg", "webpage_url_basename": "playlist", "extractor_key": "YoutubePlaylist"}
```
And
```
$ youtube-dl -J 'https://www.youtube.com/playlist?list=UU9infsKo33_2LUoiqXGgQWg'
{"_type": "playlist", "entries": [], "id": "UU9infsKo33_2LUoiqXGgQWg", "uploader": null, "uploader_id": null, "uploader_url": null, "extractor": "youtube:playlist", "webpage_url": "https://www.youtube.com/playlist?list=UU9infsKo33_2LUoiqXGgQWg", "webpage_url_basename": "playlist", "extractor_key": "YoutubePlaylist"}
```

**Comment by woefe** [comment]
I've noticed that too. It is a bug in youtube-dl's Playlist extractor for youtube.com. I have been debugging it yesterday and the extractor just doesn't return any playlist items any more. I think there was a new release today (Version 2020.11.01.1): https://yt-dl.org/download.html Haven't checked if that solves the issue, though. Will do it later today, if I find the time.

With the repository takedown, it's hard to follow bug-reports. But there is an issue reported in one of the forks that is probably related: https://github.com/l1ving/youtube-dl/issues/31

**Comment by woefe** [comment]
2020.11.01.1 was already release several days ago (I thought the .1 was new). I already have 2020.11.01.1 and so does @zoidby according to the bug-report info. I think we have to wait for a new upstream release.

**Comment by zoidby** [comment]
There is a new version: https://youtube-dl.org/

I already flagged the arch package as out-of-date.

I hope this fixes it.

**Comment by EmRowlands** [comment]
Confirmed to work on `youtube-dl-2020.11.12` on Gentoo, and `youtube-dl-2020.11.12-0` (locally built) on Artix.

**Comment by woefe** [comment]
Yes, works again with the latest release
